src/settings/NoticePolicy/NoticePolicySettings.js

```diff
@@ -80,6 +98,12 @@ class NoticePolicySettings extends React.Component {
         entryLabel={formatMessage({ id: 'ui-circulation.settings.noticePolicy.entryLabel' })}
         validate={validateNoticePolicy}
         defaultEntry={NoticePolicy.defaultNoticePolicy()}
+        isEntryInUse={this.isPolicyInUse}
```

Comment:
You need to depend on stripes-components v2.6.0, which won't be available until stripes v2.6.0 is released, to use these props. Please update the stripes dep to v2.6.0 in your {{package.json}} file. That version is in-progress on CI, so if you point your NPM `@folio:registry` at `npm-folioci` or clone {{stripes}} into your workspace, you should be all set.